packages/api/bin/comet-api-generator.js

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env node
```

Comment:
We don't need this here

Reply:
I removed this in c94ff3cacc315c168cf437a2f8d6b45a0f7c96b6